kong/templates/nginx_kong.lua

```diff
@@ -1,5 +1,6 @@
 return [[
 charset UTF-8;
+default_type '';
```

Comment:
unfortunately any configuration template change means it is breaking (for custom nginx configs) and should be opened against `next` instead...

Reply:
Also, all changed applied to `nginx_kong.lua` must be applied to the test template as well. And also, it would be better to nest this setting under the proxy `server {}` block.

Reply:
Thanks for reminding me about the test ones.

Reply:
Test template is updated. This is now rebased on `next`

Reply:
I think we should still have it moved to the proxy `server {}` block?

Reply:
I moved it to proxy `location /` block. I think it is fine like that.

Reply:
Yep, there is good!